### PR TITLE
Fix websocket protocol for demo site

### PR DESCRIPTION
**Pull request by limdauto**
## Description

The demo site is served through HTTPS so we need `wss` for websocket.

## Development notes

Lightsail does have a healthcheck for the index. The trouble is if websocket fails the server still returns an HTTP 200. We might need to look at a fix in apollo client to make it fail more gracefully and doesn't blow up the entire app.

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: src/apollo/config.js [hunk 1]
```diff
@@ -13,6 +13,8 @@ const wsHost =
     ? 'localhost:4142'
     : window.location.host;
 
+const wsProtocol = window.location.protocol === 'https:' ? 'wss' : 'ws';
```
Comment (tynandebold) [inline src/apollo/config.js:16]: Need to use this on line 19.

**Review by tynandebold — CHANGES_REQUESTED** [review]

File: src/apollo/config.js (hunk 1 above)
Comment (limdauto) [reply src/apollo/config.js:16]: Oops the file wasn't saved and commited. Nice catch.

**Review by tynandebold — APPROVED** [review]
Nice.

**Comment by limdauto** [comment]
Proof that this is working nicely: https + wss in the screenshot

Screenshot from 2022-01-26 14-53-29

File: RELEASE.md [hunk 2]
```diff
@@ -1,12 +1,8 @@
 # Upcoming release
 
-<!--
-Use the sections below to add notes for the next release.
-Please follow the established format:
-- Keep each note concise - ideally commit title length
-- Use present tense (e.g. 'Add new feature')
-- Include the ID number for the related PR (or PRs) in parentheses
--->
```
Comment (tynandebold) [inline RELEASE.md:9]: I think we should keep this in though.

**Review by idanov — APPROVED** [review]

**Pull request merged**